Terasology/MetalRenegades
# Marketplace and Currency

**To Test:**
 - Go to a marketplace, talk to the Neutral Gooey
 - A "Buy/Sell/Close" choice should come up. Each leads you to another UI to show you some items
 - Select an item and confirm the transaction.
